### DISCUSSION TITLE
Problems with updates

### DISCUSSION THREAD

**Discussion by JulieVelghe** [craftcms/cms#14997] [Q&A] [answered]
Hi, I'm trying to update a site locally, but keep running into errors:

Fout: The command "C:\php8\php.EXE "C:\Users\julie\Documents\CabinetC\Repos\Imens\storage\runtime/composer.phar" update "solspace/craft-freeform" --with-all-dependencies --working-dir C:\Users\julie\Documents\CabinetC\Repos\Imens --no-scripts --no-ansi --no-interaction" failed.

Exit Code: 2(Misuse of shell builtins)

Working directory: C:\Users\julie\Documents\CabinetC\Repos\Imens\www

Output:
================

Error Output:
================
Loading composer repositories with package information
https://composer.craftcms.com could not be fully loaded (curl error 55 while downloading https://composer.craftcms.com/p2/composer/semver.json: Connection died, tried 5 times before giving up), package information was loaded from the local cache and may be out of date
Updating dependencies
Your requirements could not be resolved to an installable set of packages.

Problem 1
- craftcms/cms 4.5.9 requires ext-intl * -> it is missing from your system. Install or enable PHP's intl extension.
- Root composer.json requires craftcms/cms 4.5.9 -> satisfiable by craftcms/cms[4.5.9].

To enable extensions, verify that they are enabled in your .ini files:
- C:\Users\Public\Documents\Appsolute\MAMPPRO\conf\php8.1.0.ini
You can also run php --ini in a terminal to see which files are used by PHP in CLI mode.
Alternatively, you can run Composer with --ignore-platform-req=ext-intl to temporarily ignore these required extensions.

Composeruitvoer: Loading composer repositories with package information
https://composer.craftcms.com could not be fully loaded (curl error 55 while downloading https://composer.craftcms.com/p2/composer/semver.json: Connection died, tried 5 times before giving up), package information was loaded from the local cache and may be out of date
Updating dependencies
Your requirements could not be resolved to an installable set of packages.

Problem 1
- craftcms/cms 4.5.9 requires ext-intl * -> it is missing from your system. Install or enable PHP's intl extension.
- Root composer.json requires craftcms/cms 4.5.9 -> satisfiable by craftcms/cms[4.5.9].

To enable extensions, verify that they are enabled in your .ini files:
- C:\Users\Public\Documents\Appsolute\MAMPPRO\conf\php8.1.0.ini
You can also run php --ini in a terminal to see which files are used by PHP in CLI mode.
Alternatively, you can run Composer with --ignore-platform-req=ext-intl to temporarily ignore these required extensions.

However, when I look at the php settings, I can see the intl extension? 
![image](https://github.com/craftcms/cms/assets/46524074/9459fea2-139b-41b2-86f1-0eb4d8f7060c)

I'm using mamp pro to run the site locally.

Any idea what could be going wrong?

**Comment by brandonkelly** [comment, marked as answer]
It’s saying that you need to recompile PHP with the `intl` extension enabled.

If you’re doing this locally, I highly recommend checking out [DDEV](https://ddev.com/) for Craft development. It’s got built-in Craft CMS support (among many others) and comes with everything Craft requires.
